cmd/aws-event-sources-controller/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.14-alpine AS builder
+
+ENV CGO_ENABLED 0
+ENV GOOS linux
+ENV GOARCH amd64
+
+WORKDIR /go/src/aws-event-sources-controller
+
+RUN apk add --no-cache openssh-client ca-certificates
```

Comment:
still trying to figure out why we want `openssh-client`. are the go modules that are downloaded with a git+ssh clone operation?